util/fetch/src/client.rs

```diff
@@ -61,6 +61,14 @@ pub trait Fetch: Clone + Send + Sync + 'static {
 		f.boxed()
 	}
 
+	/// Spawn the future in context of this `Fetch` thread pool as "fire and forget", i.e. dropping this future without
+	/// canceling the underlying future.
+	/// Implementation is optional.
+	fn forget<F, I, E>(&self, _: F) where
```

Comment:
@rphmeier You're right, it should not be optional. I think the same may be true for the `process` method.